File: Python/heartrate.py
'''
 *
 *	Problem: "Heart Rate"
 *
'''

import sys
import math

cases = input()
cases = int(cases)

index = 1
while( index <= cases):
	line = input().split()
	beats = int( line[0] )
	seconds = float( line[1] )
	
	range = 60 / seconds
	bpm = 60 * beats / seconds
	minABPM = round( (bpm - range), 4)
	maxABPM = round( (bpm + range), 4)
	
	print(minABPM, bpm, maxABPM)
	
	index += 1

File: Ruby/heartrate.rb
=begin
 *
 *	Problem:		Heart Rate
 *	Problem ID:	heartrate
 *	url:				https://open.kattis.com/problems/heartrate
 *	Solution:		Ruby
 *
=end

class Solver

	def solve
	
		cases = gets.chomp.to_i
		for this_case in 1..cases
			beats, seconds = gets.chomp.split(" ").map(&:to_f)
			
			range = 60 / seconds
			bpm = 60 * beats / seconds
			
			min_ABPM = (bpm - range).round(4)
			max_ABPM = (bpm + range).round(4)
			
			puts "#{min_ABPM} #{bpm} #{max_ABPM}"
		end
		
	end
	
end

solver = Solver.new
solver.solve
